src/Microsoft.Framework.Runtime/NuGet/NETPortable/NetPortableProfile.cs

```diff
@@ -42,27 +42,27 @@ public NetPortableProfile(string frameworkDirectory, string name, IEnumerable<Fr
         {
             if (String.IsNullOrEmpty(frameworkDirectory))
             {
-                throw new ArgumentNullException( "version");
```

Comment:
@ChengTian also this https://github.com/aspnet/dnx/blob/dev/src/Microsoft.Framework.Runtime/NuGet/NETPortable/NetPortableProfile.cs#L153 needs to be fix, I will rebase and update the PR right now, by the way I tried to build using ```dnu build``` but I faced an error in some of the dependencies

thanks ..